Applications/SimpleShot/layers/l2norm.cpp

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file   l2norm.cpp
+ * @date   09 Jan 2021
+ * @brief  This file contains the simple l2norm layer which normalizes
+ * the given feature
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug    No known bugs except for NYI items
+ *
+ */
+
+#include <iostream>
+#include <regex>
+#include <sstream>
+
+#include <nntrainer_error.h>
+#include <nntrainer_log.h>
+#include <tensor.h>
+
+#include <l2norm.h>
+
+namespace simpleshot {
+namespace layers {
+
+const std::string L2NormLayer::type = "l2norm";
+
+int L2NormLayer::initialize(nntrainer::Manager &manager) {
+  if (input_dim[0].channel() || input_dim[0].height()) {
+    ml_logw("l2norm layer is designed for channel and height is 1 for now, "
```

Comment:
If only channel and height 1 is supported, may give error than a warning?

Reply:
This will work anyway, and it might be intended so I decided to give warning.